CHANGELOG.rst

```diff
@@ -6,6 +6,65 @@
 Synapse Changelog
 *****************
 
+v2.198.0 - 2025-02-14
+=====================
+
+Automatic Migrations
+--------------------
+- Added a migration to lock any service users which are marked as archived but
+  are not locked.
+  (`#4100 <https://github.com/vertexproject/synapse/pull/4100>`_)
+- See :ref:`datamigration` for more information about automatic migrations.
+
+Model Changes
+-------------
+- See :ref:`userguide_model_v2_198_0` for more detailed model changes.
+
+Features and Enhancements
+-------------------------
+- Added brief IO yields for various LMDB operations to help improve
```

Comment:
```suggestion
- Added brief I/O yields for various LMDB operations to help improve
```

Reply:
We might need to check in about style guidance here quick.

Reply:
In the past we've used phrases like  `block the ioloop less`.

Reply:
Changed to `I/O` per conversation.